README.md

```diff
@@ -1,12 +1,23 @@
 # Palm Reading
+## Overview
 
-by [Powell's Pride Tech](./admin/team.md)
+Do you want to know what your palm lines say about you? Do you love Disney’s The Lion King? If yes, then you will enjoy our Lion King-themed palm reading app. It is a fun and interactive way to discover your destiny with the help of your favorite characters from the movie. You can chat with Simba, send them a picture or a description of your palm lines and they will tell you what they mean. You will be amazed by their insights and humor. Try it now and see for yourself.
```

Comment:
question- are we forsure making simba our persona for the palm reading app? because if we are putting it here we will need to make it consistent for the script too

Reply:
I just follow Nick's README on his branch, you can feel free to adjust the topic, or leave it as a placeholder. But we should know it this week. 
It's not like we change the whole idea with the design in the last week (next week). Then it would be a mess for all documents and UI, since right now we don't have the design system on our Figma.

Reply:
I wrote this under the impression that we were. I would be fine with making Simba the persona. I see Simba's profile picture has already been merged into the v2.0.0 branch. If we want to leave it open to having different characters, we can just not say Simba's name in the readme. Instead, we can say something more general like "You can chat with a Lion King character, ..."

Reply:
Mb, I thought we will change the whole theme but not the Lion King. Lack of sleep rn  😬

Reply:
Changed